examples/plotting/spans_and_strips.py

```diff
@@ -1,3 +1,13 @@
+''' This example demonstrates the use of ``hspan``, ``vspan``, ``hstrip`` and
+``vstrip``.
+It also demonstrates how the default ``HoverTool`` interacts with these glyphs.
```

Comment:
no need for a newline between sentences
```suggestion
``vstrip``. It also demonstrates how the default ``HoverTool`` interacts with these glyphs.
```

Reply:
I'm going to go ahead and commit this tiny change for the sake of getting this merged